jwst/lib/set_telescope_pointing.py

```diff
@@ -343,15 +347,16 @@ def update_wcs_from_fgs_guiding(model, default_pa_v3=0.0, default_vparity=1):
 
     # Get position angle
     try:
-        pa = model.meta.wcsinfo.pa_v3
+        roll_ref = model.meta.wcsinfo.roll_ref if model.meta.wcsinfo.roll_ref is not None else default_roll_ref
```

Comment:
I am missing where `default_roll_ref` is defined?

Reply:
It's an argument for the function (line 322)